_config.yml

```diff
@@ -2,7 +2,7 @@
 #
 # Use of `relative_permalinks` ensures post links from the index work properly.
 permalink:           pretty
-relative_permalinks: true
+#relative_permalinks: true
```

Comment:
If this is no longer needed, please consider removing it entirely. It's not a good practice to leave commented out code in source.